github-auth-provider/tool.gpt

```diff
@@ -0,0 +1,11 @@
+Name: GitHub
+Description: Auth provider for GitHub
+Metadata: envVars: OBOT_GITHUB_AUTH_PROVIDER_CLIENT_ID,OBOT_GITHUB_AUTH_PROVIDER_CLIENT_SECRET,OBOT_AUTH_PROVIDER_COOKIE_SECRET,OBOT_AUTH_PROVIDER_EMAIL_DOMAINS
+Metadata: optionalEnvVars: OBOT_GITHUB_AUTH_PROVIDER_TEAMS,OBOT_GITHUB_AUTH_PROVIDER_ORG,OBOT_GITHUB_AUTH_PROVIDER_REPO,OBOT_GITHUB_AUTH_PROVIDER_TOKEN,OBOT_GITHUB_AUTH_PROVIDER_ALLOW_USERS
+Credential: ../model-provider-credential as github-auth-provider
```

Comment:
Should we consider renaming this credential to something more descriptive?

Reply:
I renamed this to `placeholder-credential`